ios/Sources/AutoLayoutViewComponentView.mm

```diff
@@ -0,0 +1,80 @@
+#ifdef RCT_NEW_ARCH_ENABLED
+#import "AutoLayoutViewComponentView.h"
+#import <React/RCTConversions.h>
+#import <React/RCTViewManager.h>
+
+#import <react/renderer/components/rnflashlist/ComponentDescriptors.h>
+#import <react/renderer/components/rnflashlist/EventEmitters.h>
+#import <react/renderer/components/rnflashlist/Props.h>
+#import <react/renderer/components/rnflashlist/RCTComponentViewHelpers.h>
+
+#import "RCTFabricComponentsPlugins.h"
+#import <RNFlashList/RNFlashList-Swift.h>
```

Comment:
In our project we had to change this to:
#import <RNFlashList-Swift.h>

Or else xcode cannot find it.

Xcode 15.3
react-native 0.73.4

Reply:
```suggestion
#import <RNFlashList-Swift.h>
```